wiki/faq.md

```diff
@@ -152,10 +151,9 @@ Yes, there are currently 2 operational pools:
 ---
 ### I am crunching for the pool but the wallet still shows INVESTOR and 0 magnitude. Is that ok?
 
-That is perfectly normal when pool-mining you do work for the pool's
-account and as a reward get transactions from the pool for the work you
-did. Because of this you do not work for your own CPID and have no
-magnitude.
+On the beacon menu, make sure to click on "Pool" instead of investor or solo. Click

```

Comment:
aren't we removing "investor" verbiage?

Reply:
I was referring to the "Investor" button in the wallet, but yeah it's not needed. Simplified it to just mention the pool button